### DISCUSSION TITLE
Packages libfluidsynth2 vs libfluidsynth3

### DISCUSSION THREAD

**Discussion by codeflorist** [FluidSynth/fluidsynth#1240] [General]
On debian bookworm and ubuntu jammy+ the package is called `libfluidsynth3` (and the lib file `libfluidsynth.so.3`), while on older versions it's `libfluidsynth2` (and `libfluidsynth.so.2`).

Why is that so, if there is no major version 3 of fluidsynth anyway? Imho the package and lib file should still be named with a `2`.

This leads to the annoying problem, that due to the differently named lib file, applications built with libfluidsynth2 don't work with newer distros, which only provide libfluidsynth3, because the lib filename changed.

**Comment by derselbst** [comment]
The 2 and 3 indicate the SOVERSION, not the project version, cf https://github.com/FluidSynth/fluidsynth/issues/601#issuecomment-566712101

> Why is that so, if there is no major version 3 of fluidsynth anyway?

Because fluidsynth 2.2.0 broke ABI compatibilty, cf. https://github.com/FluidSynth/fluidsynth/wiki/ChangeLog#fluidsynth-220

**Comment by codeflorist** [comment]
Ah, ok. Thanks for the clarification!
